shared/common-adapters/input.desktop.js

```diff
@@ -153,32 +153,41 @@ class Input extends Component<void, Props, State> {
     }
   }
 
+
+
   render () {
     const underlineColor = this._underlineColor()
     const defaultRowsToShow = Math.min(2, this.props.rowsMax || 2)
     const containerStyle = this._containerStyle(underlineColor)
 
     const commonInputStyle = {
       ...globalStyles.fontSemibold,
-      fontSize: _headerTextStyle.fontSize,
-      lineHeight: `${_lineHeight}px`,
       backgroundColor: globalColors.transparent,
       color: globalColors.black_75,
       flex: 1,
       border: 'none',
       outlineWidth: 0,
       ...(this.props.small
-      ? {textAlign: 'left'}
+      ? {
+        textAlign: 'left',
+        fontSize: _bodyTextStyle.fontSize,
+        fontWeight: _bodyTextStyle.fontWeight,
+        lineHeight: _bodyTextStyle.lineHeight,
+        height: 18,
```

Comment:
Ok I moved them to `inputStyle` - I needed to set a smaller height for the `small` version since I downsized the font size